cmd/dp-observation-importer/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"github.com/ONSdigital/dp-observation-importer/config"
+	"github.com/ONSdigital/dp-observation-importer/dimension"
+	"github.com/ONSdigital/dp-observation-importer/errors"
+	"github.com/ONSdigital/dp-observation-importer/event"
+	"github.com/ONSdigital/dp-observation-importer/observation"
+	"github.com/ONSdigital/go-ns/kafka"
+	"github.com/ONSdigital/go-ns/log"
+	bolt "github.com/johnnadratowski/golang-neo4j-bolt-driver"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+)
+
+func main() {
+	log.Namespace = "dp-observation-importer"
+
+	config, err := config.Get()
+	if err != nil {
+		log.Error(err, nil)
+		os.Exit(1)
+	}
+
+	// Avoid logging the neo4j URL as it may contain a password
+	log.Debug("loaded config", log.Data{
+		"topics":                     []string{config.ObservationConsumerTopic, config.ErrorProducerTopic, config.ResultProducerTopic},
+		"brokers":                    config.KafkaAddr,
+		"bind_addr":                  config.BindAddr,
+		"import_api_url":             config.ImportAPIURL,
+		"observation_consumer_group": config.ObservationConsumerGroup,
+		"cache_ttl":                  config.CacheTTL,
+		"batch_size":                 config.BatchSize,
+		"batch_time":                 config.BatchWaitTime})
+
+	kafkaBrokers := []string{config.KafkaAddr}
+	kafkaConsumer, err := kafka.NewConsumerGroup(
+		kafkaBrokers,
+		config.ObservationConsumerTopic,
+		config.ObservationConsumerGroup,
+		kafka.OffsetNewest)
+
+	if err != nil {
+		log.Error(err, log.Data{"message": "failed to create kafka consumer"})
```

Comment:
Good spot! - The message was not appearing in the log. I have removed my custom message as it was adding no additional value.